.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -5,19 +5,95 @@ description: Suggest an idea for this project
 body:
 - type: markdown
   attributes:
-    value: |
+    value: >
+      **Thank you for wanting to suggest a feature for ansible-core!**
+
+
+
+      💡
+      Before you go ahead with your request, please first consider if it
+      would be useful for majority of the ansible-core users. As a
+      general rule of thumb, any feature that is only of interest to a
+      small sub group should be [implemented in a third-party Ansible
+      Collection][contribute to collections] or maybe even just your
+      project alone. Be mindful of the fact that the essential
+      ansible-core features have a broad impact.
+
+      If unsure, consider filing a [new proposal] instead outlining your
+      use-cases, the research and implementation considerations. Then,
+      start a discussion on one of the public [IRC meetings] we have just
+      for this.
+
+
+      <details>
+        <summary>
+          ❗ Every change breaks someone's workflow.
+        </summary>
+
+
+        [![❗ Every change breaks someone's workflow.
+        ](https://imgs.xkcd.com/comics/workflow.png)
+        ](https://xkcd.com/1172/)
+      </details>
+
+
       ⚠
-      Verify first that your issue is not [already reported on GitHub][issue search].
-      Also test if the latest release and devel branch are affected too.
-      *Complete **all** sections as described, this form is processed automatically.*
+      Verify first that your idea is not [already requested on
+      GitHub][issue search].
+
+      Also test if the devel branch does not already implement this.
+
+      *Complete **all** sections as described, this form is processed
+      automatically.*
+
+
+      **Tip:** If you are seeking community support, please consider
+      [starting a mailing list thread or chatting in IRC][ML||IRC].
+
+
+      Thank you for your collaboration!
```

Comment:
This makes sense. Although, I'm not sure how I feel about this. I think the closing thanks may refer to the call to go to the community locations instead of creating an issue here.
Maybe we could rephrase this somehow to emphasize this point instead?

FWIW if nobody has a strong opinion here, I'd like to keep this and figure out the phrasing separately. Trying to merge this ASAP and solve the controversial-but-not-terribly-bad aspects separately.

Reply:
This is an improvement over what we have, so I'm happy for this to be merged.